test/index.test.js

```diff
@@ -182,19 +232,21 @@ describe('Radar', () => {
       Navigator.getCurrentPosition.restore();
     });
 
-    it('should throw an error if no callback present', () => {
+    it('should throw an error if no callback present', (done) => {
       try {
         Radar.getLocation();
       } catch (e) {
         expect(e.message).to.equal('ERROR_MISSING_CALLBACK');
+        done();
       }
     });
 
-    it('should propagate the err if not successful', () => {
-      navigatorStub.rejects(STATUS.ERROR_LOCATION);
+    it('should propagate the err if not successful', (done) => {
+      navigatorStub.returns(Promise.reject(STATUS.ERROR_LOCATION));
```

Comment:
did `.rejects` not work here? also curious why we need `done` here but not in the `api/` files

Reply:
`rejects` implicitly wraps everything in an `Error` object, where we're just expecting them to be strings for now.

We need `done` in these tests since it's using callbacks (the api tests are all returning promises)